src/util/demangle.cpp

```diff
@@ -18,16 +18,21 @@
 
 #include "demangle.hpp"
 
+#include <regex>
+
 // Windows symbol demangler
 #ifdef _WIN32
-    // Turn off clang-format to avoid moving platform.h after Dbghelp.h
-    // (Dbghelp.h depends on types from Windows.h)
-    // clang-format off
-#    include "platform.hpp"
-#    include <Dbghelp.h>
 
-#    include <mutex>
-// clang-format on
+    // Dbghelp.h depends on types from Windows.h so it needs to be included first
```

Comment:
You mention Windows.h but don't actually include it, you should probably mention that platform.hpp includes it?

Reply:
Mentioned platform.hpp includes windows